Examples/Network Sample/README.md

```diff
@@ -0,0 +1,4 @@
+### Network sample
+
+This example shows the URLSessionInstrumentation in action. 
+It only shows the minimum configuration: Just by initializing the class, all network request that happen after it will be captured by Opentelemetry as Spans. Check URLSessionInstrumentation README for more information.
```

Comment:
```suggestion
It only shows the minimum configuration: Just by initializing the class, all network request that happen after it will be captured by Opentelemetry as Spans. Check [`URLSessionInstrumentation` README](Sources/Instrumentation/URLSession/README.md ) for more information.
```